packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -548,3 +553,27 @@ function moveLiToList(li: HTMLElement) {
         unwrap(li.parentNode!);
     }
 }
+
+function createListStyleMetadata(
+    style: ListStyleMetadata | null,
+    listType: ListType | CompatibleListType,
+    orderedStyle: NumberingListType,
+    unorderedStyle: BulletListType
+): ListStyleMetadata {
+    if (style) {
+        if (listType === ListType.Ordered && orderedStyle !== undefined) {
+            style.orderedStyleType = orderedStyle;
+        } else if (unorderedStyle !== undefined) {
+            style.unorderedStyleType = unorderedStyle;
+        }
```

Comment:
Can we just update the style without checking list type? So that means even for a ordered list, it is still allowed to updated unordered style, as long as the value is not undefined